packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -211,7 +211,9 @@ const AppointmentsForm: React.FC<AppointmentsFormProps & DefaultWorkspaceProps>
 
   const defaultDateAppointmentScheduled = appointment?.dateAppointmentScheduled
     ? new Date(appointment?.dateAppointmentScheduled)
-    : new Date();
+    : appointment?.endDateTime
+      ? new Date(appointment?.endDateTime)
+      : new Date();
```

Comment:
Why are we setting the `dateAppointmentScheduled` to appointment endDatetime. Its seems the API response isn't saving and showing `dateAppointmentScheduled` we should update the version of appointment module version to 2.1.0-SNAPSHOT  [here](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/3ee096071415feadfe2f61df5a548d1df1a33908/distro/pom.xml#L39)